src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -688,29 +688,30 @@ void AtlasEngine::_resolveFontMetrics(const wchar_t* requestedFaceName, const Fo
     const auto underlineWidth = std::max(1.0f, std::roundf(underlineThickness));
     const auto strikethroughPos = std::roundf(baseline + strikethroughPosition);
     const auto strikethroughWidth = std::max(1.0f, std::roundf(strikethroughThickness));
-    const auto thinLineWidth = std::max(1.0f, std::roundf(underlineThickness / 2.0f));
+    const auto doubleUnderlineWidth = std::max(1.0f, std::roundf(underlineThickness / 2.0f));

```

Comment:
`fontMetrics->doubleUnderline[2]` also needs to be updated to store `doubleUnderlineWidth` (actually, `doubleUnderlineWidthU16`).

Currently, it's using `thinLineWidth`, so double/curly underlines look _bolder_ than normal.